multiqc/modules/nonpareil/nonpareil.py

```diff
@@ -0,0 +1,276 @@
+        # y.p75 Percentile 75 (3rd quartile) of rarefied coverage
+
+        import numpy as np
+
+        for s_name, data in json_raw.items():
+            # Conbert base pairs to gigabase pairs
+            data["LR"] = data["LR"] / 1000 / 1000
+            data["LRstar"] = data["LRstar"] / 1000 / 1000
+            data["x.adj"] = [(1e-6 if x == 0 else x) / 1000 / 1000 for x in data["x.adj"]]
+            data["x.model"] = [x / 1000 / 1000 for x in data["x.model"]]
+            data["observed"] = {x: y for x, y in zip(data["x.adj"], data["y.cov"])}
+            data["model"] = {x: y for x, y in zip(data["x.model"], data["y.model"])}
+            # Calculate dispersion
+            # from https://github.com/lmrodriguezr/nonpareil/blob/162f1697ab1a21128e1857dd87fa93011e30c1ba/utils/Nonpareil/R/Nonpareil.R#L306-L318
+            disp_add = False
+            if self.disp_type == "sd":
+                disp_add = data["y.sd"]
+            elif self.disp_type == "ci95":
+                disp_add = list(np.array(data["y.sd"]) * 1.9)
+            elif self.disp_type == "ci90":
+                disp_add = list(np.array(data["y.sd"]) * 1.64)
+            elif self.disp_type == "ci50":
+                disp_add = list(np.array(data["y.sd"]) * 0.67)
+            elif self.disp_type == "iq":
+                data[f"{self.disp_type}_upper"] = {x: y for x, y in zip(data["x.adj"], data["y.p75"])}
+                data[f"{self.disp_type}_lower"] = {x: y for x, y in zip(data["x.adj"], data["y.p25"])}
+
+            if disp_add:
+                data[f"{self.disp_type}_upper"] = {
+                    x: y for x, y in zip(data["x.adj"], list(np.array(data["y.cov"]) + np.array(disp_add)))
+                }
+                data[f"{self.disp_type}_lower"] = {
+                    x: y for x, y in zip(data["x.adj"], list(np.array(data["y.cov"]) - np.array(disp_add)))
+                }
+
+        return json_raw
+
+    def nonpareil_general_stats_table(self):
+        """Take the parsed stats from the nonpareil report and add it to the
+        General Statistics table at the top of the report"""
+
+        headers = {
+            "L": {
+                "title": "Read Length",
+                "min": 0,
+                "suffix": " bps",
+                "scale": "GnBu",
+                "format": "{:,.3f}",
+                "hidden": True,
+            },
+            "AL": {
+                "title": "Adjusted read length",
+                "description": "Adjusted read length (if using kmers)",
+                "min": 0,
+                "scale": "GnBu",
+                "suffix": " bps",
+                "format": "{:,.3f}",
+                "hidden": True,
+            },
+            "R": {
+                "title": "Number of reads",
+                "min": 0,
+                "scale": "GnBu",
+                "format": "{:,.0f}",
+                "hidden": True,
+            },
+            "LR": {
+                "title": "Sequencing effort",
```

Comment:
Same as the read count multiplier above (https://multiqc.info/docs/development/compatibility/#updated-number-formatting), but for base count (e.g. `config.base_count_prefix`, etc).